ern-container-gen-android/src/AndroidGenerator.ts

```diff
@@ -707,7 +691,49 @@ You should replace "${annotationProcessorPrefix}:${dependency}" with "annotation
           'lib/src/main/java/com/walmartlabs/ern/container/plugins',
         );
         shell.cp(pathToPluginConfigHook, pathToCopyPluginConfigHookTo);
+
+        this.updatePluginsToAndroidxImports(
+          rnVersion,
+          plugin,
+          pathToCopyPluginConfigHookTo,
+        );
+      }
+    }
+  }
+
+  /**
+   * Modify Plugins to support Androidx
+   * eg: import android.support.annotation.nonNull => import androidx.annotation.nonNull
+   */
+  public updatePluginsToAndroidxImports(
```

Comment:
Thoughts about a more accurate name for this function? How about `convertToAndroidX`?